themes/src/main/resources/theme/base/admin/resources/js/controllers/clients.js

```diff
@@ -1299,6 +1299,14 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
             }
         }
 
+        if ($scope.client.attributes["backchannel.user.code.parameter.required"]) {
+            if ($scope.client.attributes["backchannel.user.code.parameter.required"] == "true") {
```

Comment:
Could you follow what I commented on `OIDCConfigAttributes`?

Reply:
Changed this, thank you.